no-computer.md

```diff
@@ -6,39 +6,47 @@ This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
 
+
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Grading a problem set for a math or science class can be tricky and subjective. How do you determine how many points an incorrect solution gets? Graders have to use inuition to determine how close they are to a solution. Rubrics are essentially a DSL for grading, as they remove some of the subjectivity of this process. With rubrics, someone simply references the part of the problem set on the rubric, and reads off the points to give. 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+The rubric itself is a simple language, only specifyin the amount of points to give someone for various different types of answers. Its a language because it has its own grammar rules of how to specify parts of a problem, breaking it down, and giving each a point value. It addresses the users needs by removing the subjective element of decision making. 
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I grade classes. In Algs grading, we always create a detailed rubric for each problem, so we don't have to carefully read each part of the problem, but can just look for the parts we care about and input the corresponding grades. 
 
 ### Domain
 _Describe the project's domain in five words._
 
+Problem set grading breakdown system
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user interacts essentially with a table of problem part to point values. The programming language is the rubric, and a program is the problem set. To program is to map the parts of the problem set to the rubric. This is the simplest way to interact with the domain, because it breaks each problem into logical parts, so if someone answers part of the problem, but not all, you can pick out which parts they did and give the corresponding points.
 
```

Comment:
Okay, I'd buy it.  An issue with this DSL is the amount of overhead before you can start grading.  You need to have a certain number of students before it becomes worth it, and sometimes that number can be quite high.